index.js

```diff
@@ -1,18 +1,30 @@
+require("dotenv").config();
```

Comment:
Please remove this line. The "start" script on package.json pre-loads the environment variables so is not necessary to do this.